help/create-a-stream.md

```diff
@@ -36,11 +36,12 @@ There are several parameters you can set while creating a stream. All but
 * **Stream description**: Helps users decide whether to subscribe when they
   are browsing streams.
 
-* **Who can access the stream?**: See [Stream permissions](/help/stream-permissions).
+* **Announce new stream in #<stream configured by your organization\>**: Posts
+  a message to stream configured by your organization for `New stream annoucements`
+  advertising the new stream. Organization administrators can change the stream
```

Comment:
Hm, let's try making the explanation:

> Posts a message in the "new streams" topic advertising the new stream. Organization administrators... (etc.)

Reply:
@drrosa Do you have any suggestions here regarding how to mention the to-be-specified steam in the bold part of this bullet? I'm not sure if have a better approach than the `< ... >` formatting.

Reply:
Any other feedback here?

Reply:
Hmm, how about:

> - **Announce new stream in the [stream configured by your organization](/help/configure-notification-bot#new-stream-announcements)**: Posts a message in the "new streams" topic advertising the new stream. 

and we can drop the last sentence: "Organization administrators can change the stream used for stream creation announcements."

Reply:
Sure, that works for me.